src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIStructure.java

```diff
@@ -1046,6 +1046,16 @@ else if (block instanceof BlockFlowerPot)
             }
         }
 
+        if(block instanceof BlockChest && job instanceof JobBuilder)
+        {
+            BlockPos buildingLocation = ((JobBuilder) job).getWorkOrder().getBuildingLocation();
+            AbstractBuilding building = this.getOwnBuilding().getColony().getBuilding(buildingLocation);
+            if(building instanceof AbstractBuilding)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless "instanceof" operator; it will always return "true". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1850)